.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,6 +1,6 @@
 name: Bug Report
 description: Report Rancher Desktop issue
-title: "[Bug]- "
+title: " "
```

Comment:
Can't this be the empty string? Would the new setting prefix every title with a space?

Reply:
Leaving it empty or without space throws an error:
![Screen Shot 2021-11-17 at 12 39 07 PM](https://user-images.githubusercontent.com/37411123/142262024-59426bbd-04f1-4dd7-ad45-ab9920cfa879.png)
I think it must
![Screen Shot 2021-11-17 at 12 43 50 PM](https://user-images.githubusercontent.com/37411123/142262658-c2f24666-e2d8-4a63-93aa-24e9a03365c7.png)

Best solution: remove the entire tag 😄